qcfractal/storage_sockets/storage_socket.py

```diff
@@ -0,0 +1,50 @@
+"""
+A factory for Database sockets
+"""
+
+
+def storage_socket_factory(url,
```

Comment:
Try to remember `git mv` if you can. Saves drastically on the history.

Reply:
PyCharm should take care of all that sort of refactoring. I wonder if its because I had to resolve the diffs between the services->procedure PR and so the `git mv` would not work correctly.